sykle/plugins/dj/__init__.py

```diff
@@ -0,0 +1,89 @@
+"""dj
+
+Usage:
+  syk dj new-project <name> \
+    [--apps=<apps>] \
+    [--api-framework=<api_framework>] \
+    [--cms-framework=<cms_framework>] \
+    [--email-backend=<email_backend>] \
+    [--error-logging=<error_logging] \
+    [--storage-backend=<storage_backend>] \
+    [--search-backend=<search_backend>] \
+    [--task-queue=<task_queue]
+
+Options:
+  --apps=<apps>                         A comma-separated list of the apps to
+                                        stub. Defaults to ["core"].
+  --api-framework=<rest,graphql>        An API framework to install.
+  --cms-framework=<wagtail>             A CMS framework to install.
+  --email-backend=<ses>                 An email backend to install.
+  --error-logging=<sentry>              An error logging service to install.
+  --storage-backend=<s3>                A storage backend in which to store
+                                        media and static files.
+  --search-backend=<elasticsearch_dsl>  A backend for textual filtering.
+  --task-queue=<celery>                 An backend for asynchronous tasks.
+  -h --help                             Show this screen.
+
+Description:
+  new-project                           Create a django project.
+"""
+
+__version__ = '0.1.0'
+import logging
+
+from docopt import docopt
+
+from sykle.plugin_utils import IPlugin
+
+from .template_renderer import TemplateRenderer
+from .utils import kebab_arg_to_snake_case
+
+
+logger = logging.getLogger(__name__)
+
+
+class Plugin(IPlugin):
+    NAME = 'dj'
```

Comment:
Unless I'm missing something, I'm pretty sure this plugin will come in and override the `dj` alias for `django-admin` we're defining in a lot of our projects. Think we should change this to `boilerplate` or just `new-project` (minus the extra argumenet) regardless of whether this will fall through to django admin or override it, as this is independent of django admin